src/styles/main.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  --column-count: 1;
+
+  min-width: 1024px;
+  max-width: 1200px;
+  padding: 50px 40px 72px;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
```

Comment:
The `grid-template-columns` property uses a CSS variable for the number of columns, which is good. However, ensure that the `200px` value for the column width is consistent with the design requirements. If it needs to be flexible, consider using `1fr` as a unit or another relative unit instead of a fixed pixel value.